.github/workflows/kfp-kubernetes-execution-tests.yml

```
@@ -31,6 +31,11 @@ jobs:
         with:
           python-version: '3.9'
 
+      - name: Free up space in /dev/root
```

Do you mind adding a comment with the link here? Something like the following should be enough.

```suggestion
      # This is intended to address disk space issues that have surfaced intermittently during CI - https://github.com/actions/runner-images/issues/2840#issuecomment-1284059930
      - name: Free up space in /dev/root
```